### DISCUSSION TITLE
Multiple coil image recontruction

### DISCUSSION THREAD

**Discussion by Stockless** [MagneticResonanceImaging/MRIReco.jl#208] [Q&A] [answered]
Hi! I am implementing multiple coil simulations in KomaMRI. In the raw data profile I added one column per coil, and when using MRIReco.jl I get one image per coil. I was wondering how would I instead reconstruct an image with all coil data combined? I saw the example in [X](https://magneticresonanceimaging.github.io/MRIReco.jl/stable/SENSE/), but I was not able to make it work. Thanks!
![Screenshot from 2024-12-10 18-08-34](https://github.com/user-attachments/assets/43432cc0-6008-4dae-9f25-039d28fe7690)

**Comment by aTrotier** [comment]
Most of the documentation is outdated... 

The interface for solving convex problems has changed. Can you try with :

```julia
# reco parameters
params = Dict{Symbol, Any}()
params[:reco] = "multiCoil"
params[:reconSize] = (N,N)
  params[:reg] = L2Regularization(1.e-3)       # regularization
params[:iterations] = 40
params[:solver] = CGNRE
params[:senseMaps] = coilsens

# do reconstruction
Ireco = reconstruction(acqData, params)
```

If it still does not works, can you share the error.

**Reply by Stockless** [reply, marked as answer]
It worked!
I made 2 small changes in the code, replacing `params[:solver] = CGNRE` with `params[:solver] = CGNR`, and adding `using MRIReco.RegularizedLeastSquares`, and it reconstructed the multiple-coil image correctly. Thank you!
![MRIReco](https://github.com/user-attachments/assets/d9e7cf70-555f-4925-810c-25ae4b2feec9)
